independent-publisher-connectors/PowerBI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1058 @@
+{

```

Comment:
@paulculmsee There is anther PR open for same connector, If it is not different, please remove this connector files from this PR